packages/messaging/index.js

```diff
@@ -97,10 +97,14 @@ export class Messaging {
      *
      * @param {string} name
      * @param {Record<string, any>} [data]
+     * @param {object} [overrides]
+     * @param {string} [overrides.id] - the ID sent with outgoing requests
      * @return {Promise<any>}
      */
-    request (name, data = {}) {
-        const id = globalThis?.crypto?.randomUUID?.() || name + '.response'
+    request (name, data = {}, options = {}) {
```

Comment:
Not sure why TS isn't whining about this:

```suggestion
    request (name, data = {}, overrides = {}) {
```